sdk/iot/hub/src/az_iot_hub_client_twin.c

```diff
@@ -12,15 +12,17 @@
 #include <_az_cfg.h>
 
 static const uint8_t az_iot_hub_client_twin_hashtag = '#';
-static const az_span az_iot_hub_client_twin_request_id_suffix = AZ_SPAN_LITERAL_FROM_STR("?$rid=");
-
-static const az_span az_iot_hub_twin_response_sub_topic
-    = AZ_SPAN_LITERAL_FROM_STR("$iothub/twin/res/");
-static const az_span az_iot_hub_twin_get_pub_topic = AZ_SPAN_LITERAL_FROM_STR("$iothub/twin/GET/");
+static const uint8_t az_iot_hub_client_twin_question = '?';
```

Comment:
nit: Also leaving comment that we later need to consolidate all these in a separate file, to avoid repeated vars throughout the SDK.

Reply:
Added #605